ml-agents-envs/mlagents/envs/side_channel/float_properties_channel.py

```diff
@@ -52,7 +52,14 @@ def list_properties(self) -> List[str]:
         Returns a list of all the string identifiers of the properties
         currently present in the Unity Environment.
         """
-        return self._float_properties.keys()
+        return list(self._float_properties.keys())
+
+    def get_property_dict(self) -> Dict[str, float]:
```

Comment:
Add this since the usage pattern was always list_properties then get_property on each.

Reply:
Do we still need get and list properties then?
Also, it is not clear that modifying this dict will not modify the unity environment...

Reply:
Not sure we need list. I think get() is still useful. 

Would `get_property_dict_copy()` make it clearer? Unfortunately there's no clean way to return a read-only dictionary.